internal/resolvers/youtube/model.go

```diff
@@ -9,3 +9,10 @@ type youtubeTooltipData struct {
 	LikeCount    string
 	DislikeCount string
 }
+
+type youtubeChannelTooltipData struct {
+	Title        string
+	PublishDate  string
```

Comment:
Here you call this property `PublishDate`, however in the `youtubeChannelTooltip` its "label" is named `Published Date` which isn't quite consistent. Please choose one of either and stick with it.

Reply:
Will align to `PublishedDate`